src/applications/edu-benefits/10203/pages/activeDuty.js

```diff
@@ -0,0 +1,31 @@
+import fullSchema1995 from 'vets-json-schema/dist/22-1995-schema.json';
+import { housingPaymentInfo } from '../content/activeDuty';
+import _ from 'lodash';
+
+const { isActiveDuty } = fullSchema1995.properties;
+
+export const uiSchema = {
+  isActiveDuty: {
+    'ui:title':
+      'Are you currently on active duty or do you anticipate you will be going on active duty?',
+    'ui:widget': 'yesNo',
+  },
+  'view:housingPaymentInfo': {
+    'ui:description': housingPaymentInfo,
+    'ui:options': {
+      hideIf: data => !_.get(data, 'isActiveDuty', false),
```

Comment:
You might be able to use optional chaining here - I haven't thoroughly tested the logic:

```suggestion
      hideIf: data => !data?.isActiveDuty,
```